client/webpack.config.dev.js

```diff
@@ -26,6 +26,8 @@ module.exports = {
           G_SUITE_DOMAIN: JSON.stringify(process.env.G_SUITE_DOMAIN),
           CLIENT_ID: JSON.stringify(process.env.CLIENT_ID),
           API_URI: JSON.stringify(process.env.REACT_APP_API_URI),
+          ANDELA_API_BASE_URL: JSON.stringify(process.env.ANDELA_API_BASE_URL),
```

Comment:
You know what, on second thought, we can leave this here.

Reply:
Though I feel some variables like the BASE URL aren't sensitive like that so they can live in a config object.